<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrimmerGUI</class>
 <widget class="QFrame" name="TrimmerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrimmerGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border-top: 2px solid #444;
	border-left: 2px solid #444;
	border-right: 2px solid #444;
}

QFrame#videoFrame
{
	border-width: 2px;
	border-radius: 0px;
	background: none;
	border-style: solid;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="videoFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>772</width>
     <height>436</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border: 2px solid #444;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="PreviewerRGB" name="preview" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>768</width>
      <height>432</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelClipName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No Clip Loaded</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="framePlayControls">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>886</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#framePlayControls
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #AAA, stop: 1 #777);
	border: 2px solid #444;
}

QLabel
{
	color: white;
	border: none;
	background:none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSlider" name="sliderSeek">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>0</y>
      <width>861</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNextFrame">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/nextframe.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/nextframe_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPreviousFrame">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/previousframe.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/previousframe_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForward">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/forward.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/forward_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReverse">
    <property name="geometry">
     <rect>
      <x>323</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/reverse.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/reverse_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlayPause">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/play.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/play_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenFile">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>75</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/open_file.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/open_file_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelClipDuration">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelClipTimecode">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDash">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>40</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAspectRatio">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>75</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/4x3.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/4x3_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameTrimControls">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>886</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frameTrimControls
{
	background: #444;
	border: 2px solid #333;
}

QLabel
{
	border-radius: 10px;
	background: #111;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonGoToOut">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>55</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/gotoout.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/gotoout_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMarkOut">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>27</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/markout.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/markout_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMarkIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>27</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/markin.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/markin_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGoToIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Goto In</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/gotoin.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/gotoin_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrimDuration">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>11</y>
      <width>142</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClipOut">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>11</y>
      <width>135</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>--:--:--:--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClipIn">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>11</y>
      <width>135</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>--:--:--:--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMarkClip">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>10</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/markclip.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/markclip_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearMarks">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>10</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/clearmarks.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/clearmarks_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExtract">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>52</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/extract.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/extract_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>62</y>
      <width>731</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
  </widget>
  <widget class="QVUMeter" name="audioMeter" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>101</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewerRGB</class>
   <extends>QWidget</extends>
   <header>previewerrgb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVUMeter</class>
   <extends>QWidget</extends>
   <header>qvumeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
